cluster-autoscaler/cloudprovider/clusterapi/clusterapi_nodegroup.go

```diff
@@ -79,11 +83,16 @@ func (ng *nodegroup) IncreaseSize(delta int) error {
 	if delta <= 0 {
 		return fmt.Errorf("size increase must be positive")
 	}
-	size := int(ng.scalableResource.Replicas())
-	if size+delta > ng.MaxSize() {
-		return fmt.Errorf("size increase too large - desired:%d max:%d", size+delta, ng.MaxSize())
+
+	size, err := ng.scalableResource.Replicas()
+	intSize := int(size)
+	if err != nil {
+		return err
 	}
```

Comment:
nit: Don't need to declare intSize until the error is checked
```suggestion
	size, err := ng.scalableResource.Replicas()
	if err != nil {
		return err
	}
	
	intSize := int(size)
```